packages/dashboard/src/app/routes/_authenticated/_products/components/create-product-variants.tsx

```diff
@@ -262,16 +265,12 @@ export function CreateProductVariants({
                                                 render={({ field }) => (
                                                     <FormItem>
                                                         <FormControl>
-                                                            <div className="relative">
-                                                                <span className="absolute left-3 top-2.5">
-                                                                    {currencyCode}
-                                                                </span>
-                                                                <Input
+                                                                <MoneyInput
                                                                     {...field}
-                                                                    className="pl-12"
-                                                                    placeholder="0.00"
+                                                                    value={Number(field.value) || 0}
+                                                                    onChange={value => field.onChange(value.toString())}
+                                                                    currency={activeChannel?.defaultCurrencyCode ?? 'USD'}
```

Comment:
```suggestion
currency={activeChannel?.defaultCurrencyCode}                                       
```

The money input itself has a fallback to the `activeChannel`'s default currency. I think we can either remove the USD string or remove this line entirely so that we don't need to fall back to the USD string in multiple places. I believe that by default USD is the channel's default currency from looking through the codebase, and it might be safer to rely on other components to handle this. What do you think @michaelbromley?